concourse/pipelines/deployment-kick-off.yml

```diff
@@ -82,3 +82,32 @@ jobs:
                 echo "Pipeline kick-off is enabled. Updating. (set ENABLE_MORNING_DEPLOYMENT=false to disable)"
 
                 make -C ./paas-cf "${AWS_ACCOUNT}" trigger-deploy
+
+  - name: pause-kick-off
+    serial: true
+    plan:
+      - get: deployment-timer
+        trigger: true
+      - get: paas-cf
+      - task: pause-kick-off
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/self-update-pipelines
+              tag: 0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2
+          inputs:
+            - name: paas-cf
+          params:
+            AWS_ACCOUNT: ((aws_account))
+            AWS_DEFAULT_REGION: ((aws_region))
+            DEPLOY_ENV: ((deploy_env))
+            SKIP_AWS_CREDENTIAL_VALIDATION: true
```

Comment:
I see why using these three AWS variables is necessary, but it is starting to feel weird how dependent this is on AWS, given all it needs to do is get the Concourse ATC password from an S3 bucket (for which it can use its own instance profile).

No action required here, just thought I would comment because it is weird.